_posts/2021-02-01-launching-the-solidity-forum.md

```diff
@@ -0,0 +1,77 @@
+
+It will **not** be the place for…
+
+- Ad-hoc or specific support questions. For urgent Solidity support questions, please use the [Solidity Gitter chat](https://gitter.im/ethereum/solidity) or consider checking out the [Ethereum StackExchange](https://ethereum.stackexchange.com/).
+- Solidity bugs, vulnerabilities or issues. To report a bug, please use the [GitHub issues tracker](https://github.com/ethereum/solidity/issues) and refer to this guide on [how to report issues](https://docs.soliditylang.org/en/latest/contributing.html#how-to-report-issues).
+- Generic Ethereum discussion. For that visit [r/ethereum](https://www.reddit.com/r/ethereum).
+- Ethereum research specific discussion. For that, please refer to the [ethresear.ch](https://ethresear.ch/) forum.
+- EIP specific discussion. For that visit the [Ethereum Magicians](https://ethereum-magicians.org/) forum.
+
+The Solidity forum is based on [Discourse](https://www.discourse.org/about), an open-source software for civilized discussions. :)
+
+If you are new to Discourse consider having a look at this [“Unofficial Discourse User Reference Guide”](https://forum.digikey.com/t/an-unofficial-discourse-user-reference-guide/1125), which summarizes useful tips on how to use Discord, text formatting, images, LaTeX and more.
+ 
+### A Quick Guide to the Solidity Forum Categories 🔖
+
+We are launching the forum with an initial selection of categories which can be adjusted and extended over time. To propose a new category, or the change of an existing one, please post your suggestions in the [“Site Feedback”](https://forum.soliditylang.org/c/site-feedback/2) category. 
+
+#### → Announcements
+[”Announcements”](https://forum.soliditylang.org/c/announcements/5) is a low-traffic category for important announcements about the Solidity language and compiler.
+
+📮 Subscribe to this category if you want to be kept in the loop about releases, security updates and Solidity-relevant feedback surveys, news and events.
+
+#### → Solidity Compiler
+[“Solidity Compiler”](https://forum.soliditylang.org/c/solidity-compiler/6) is a category for discussions and feedback about the Solidity compiler.
+
+⚠️ Please do not use the forum to report issues. Please report issues directly via the [Solidity Issue Tracker](https://github.com/ethereum/solidity/issues/new/choose) in Github.
+
+#### → Language Design
+[“Language Design”](https://forum.soliditylang.org/c/language-design/9) is the dedicated place for proposals and discussion on new language features and their implementation in the early stages of ideation. This is the place to discuss the soundness of proposals and define them further. As soon as proposals get more tangible their implementation will also be discussed in the Solidity GitHub organisation in the form of GitHub issues.
+
+Examples of topics that could fit the Language Design category:
+
+- "Change the way inheritance works"
+- "Introduce a ``switch`` statement" 
+
```

Comment:
Should we also mention something like "A vague \"I don't like how X is working\" might get a discussion started, but the more specific solutions an entry proposes, the more likely it is to be adopted into the language"?

Reply:
(Just thinking about my bad example above)